### ISSUE TITLE
Checklist Point

### ISSUE THREAD

**Issue by joelworsham** [realbig/playbook#6]
Site build out checklist.
# UNCHECK DISABLE SITE FROM BEING INDEXED

**Comment by BigActual** [comment]
So, I had a question about this. Our dev sites are publicly accessible in many instances. The ping service behind the system this check box addresses will bring in crawlers, I believe, as we've had dev sites indexed in the past. Wouldn't it be better to uncheck this box at launch, and or put the site behind a Pardon our dust page for non-logged in users?

**Comment by joelworsham** [comment]
Yes, my idea with this checklist point is for launch. It would probably also be a good idea for us to be more in the habbit of password protecting our staging sites.
